recipes/metpy/meta.yaml

```diff
@@ -0,0 +1,41 @@
+package:
+  name: metpy
+  version: !!str 0.3.0
+
+source:
+  fn: metpy-0.3.0.tar.gz
+  url: https://pypi.python.org/packages/source/M/MetPy/MetPy-0.3.0.tar.gz
+  md5: efa24aea506b2d0a829035aea94e6d89
+
+build:
+    script: python setup.py install --single-version-externally-managed --record record.txt
+    number: 0
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+# Specify numpy with >= to get it to name packages without numpy dependency
```

Comment:
That has been fixed for a few versions of conda-build. If you have no compiled dependency, just specifying ```numpy``` will be fine